test/fixtures/generated/default/ubuntu1610-32l

```diff
@@ -0,0 +1,21 @@
+---
+arguments_string: ubuntu1610-32l
+environment_variables: {}
+expected_hash:
+  HOSTS:
+    ubuntu1610-32-1:
+      pe_dir: 
+      pe_ver: 
+      pe_upgrade_dir: 
+      pe_upgrade_ver: 
+      hypervisor: vmpooler
+      platform: ubuntu-16.10-i386
+      template: ubuntu-1610-i386
+      roles:
+      - agent
+      - classifier
+  CONFIG:
+    nfs_server: none
+    consoleport: 443
+    pooling_api: http://vmpooler.delivery.puppetlabs.net/
+expected_exception: 
```

Comment:
Is this boilerplate or is an entry for 32-bit systems necessary? I didn't realize we were still testing 32-bit *NIX platforms.